glean-core/python/tests/test_glean.py

```diff
@@ -322,7 +337,7 @@ def test_ping_collection_must_happen_after_currently_scheduled_metrics_recording
     ping.submit()
 
     # Wait until the work is complete
-    Dispatcher._task_worker._queue.join()
+    # Dispatcher._task_worker._queue.join()
```

Comment:
Another commented out line that I'm not sure if it was intentionally left or not.